torch_geometric/nn/models/__init__.py

```diff
@@ -10,7 +10,6 @@
 from .schnet import SchNet
 from .dimenet import DimeNet, DimeNetPlusPlus
 from .explainer import Explainer, to_captum
-from .gnn_explainer import GNNExplainer
```

Comment:
Can we maintain some form of backward compatibility for the old interface. I don't want existing scripts from users to break here :(

Reply:
I added a `GNNExplainer_` that should do the trick in [22a3376](https://github.com/pyg-team/pytorch_geometric/pull/5967/commits/22a33764a16b4327390428d8ab8377def1fd4ea8). No idea how to actually format it and where to put it.

Reply:
For full backward compatibility we should put `GNNExplaienr_` in `nn/models/gnn_explainer` and call it `GNNExplainer` with a depreciation warning, then we can remove it in a while.

Reply:
thanks, I'll do that